cfme/tests/control/test_actions.py

```diff
@@ -246,113 +202,54 @@ def policy_profile_name(name_suffix):
     """ Ensures that the VM is off when the control goes to the test."""
     vm.wait_vm_steady()
-    if vm.is_vm_suspended():
+    if provider.one_of(InfraProvider, AzureProvider, OpenStackProvider) and vm.is_vm_suspended():
         vm.start_vm()
         vm.wait_vm_running()
     if not vm.is_vm_stopped():
         vm.stop_vm()
         vm.wait_vm_stopped()
     # Make sure the state is consistent
-    vm_crud.refresh_relationships(from_details=True)
-    vm_crud.wait_for_vm_state_change(desired_state=vm_crud.STATE_OFF, from_details=True)
+    vm.crud.refresh_relationships(from_details=True)
+    vm.crud.wait_for_vm_state_change(desired_state=vm.crud.STATE_OFF, from_details=True)
     return vm
 
 
-@pytest.fixture(scope="function")
-def vm_crud_refresh(vm_crud, provider):
-    """Refreshes the VM if that is needed for the provider."""
-    if provider.type in {"ec2"}:
-        return lambda: vm_crud.refresh_relationships(from_details=True)
-    else:
-        return lambda: None
-
-
-@pytest.yield_fixture(scope="module")
-def policy_for_testing(automate_role_set, vm, policy_name, policy_profile_name, provider):
-    """ Takes care of setting the appliance up for testing """
-    policy = policies.VMControlPolicy(
-        policy_name,
-        scope="fill_field(VM and Instance : Name, INCLUDES, {})".format(vm.name)
-    )
-    policy.create()
-    policy_profile = policy_profiles.PolicyProfile(policy_profile_name, policies=[policy])
-    policy_profile.create()
-    yield policy
-    policy_profile.delete()
-    policy.delete()
-
-
-@pytest.yield_fixture(scope="module")
-def policy_for_testing_for_vm_big(
-        automate_role_set, vm_big, policy_name, policy_profile_name, provider):
-    """ Takes care of setting the appliance up for testing """
-    policy = policies.VMControlPolicy(
-        policy_name,
-        scope="fill_field(VM and Instance : Name, INCLUDES, {})".format(vm_big.name)
-    )
-    policy.create()
-    policy_profile = policy_profiles.PolicyProfile(policy_profile_name, policies=[policy])
-    policy_profile.create()
-    yield policy
-    policy_profile.delete()
-    policy.delete()
-
-
-@pytest.yield_fixture(scope="module")
-def assign_policy_for_testing(vm, policy_for_testing, provider, policy_profile_name):
-    provider.assign_policy_profiles(policy_profile_name)
-    yield policy_for_testing
-    provider.unassign_policy_profiles(policy_profile_name)
-
-
-@pytest.yield_fixture(scope="module")
-def assign_policy_for_testing_vm_big(
-        vm_big, policy_for_testing_for_vm_big, provider, policy_profile_name):
-    provider.assign_policy_profiles(policy_profile_name)
-    yield policy_for_testing_for_vm_big
-    provider.unassign_policy_profiles(policy_profile_name)
-
-
-def test_action_start_virtual_machine_after_stopping(
-        request, assign_policy_for_testing, vm, vm_on, vm_crud_refresh):
+@pytest.mark.uncollectif(lambda provider: not provider.one_of(VMwareProvider, RHEVMProvider,
```

Comment:
Instead of this uncollectif group of providers on each test just to filter out GCE+SCVMM, why not only generate for these  providers?